app/orgs/[orgId]/usage/page.tsx

```diff
@@ -0,0 +1,120 @@
+import { PageHeader } from '@/components/PageHeader';
+import { getOrgUsagePage } from '@/controllers/controllers';
+import { formatDate } from '@/helpers/dates';
+
+function formatCost(plan: any): string {
```

Comment:
[note] I smushed multiple components and associated functions into this single page for now, since the UI was not the primary goal of this exploratory mission.  I assume most everything about it will be changing, and decided it would be easier to delete one file rather than chase down multiple component files and helper functions in the future.  I'm happy to break them up, however, is that's the preference!